### PR TITLE
Add text_area helper to form_for

### PR DESCRIPTION
**Pull request by khaiql**
cc @AlfonsoUceda, @jodosha

### CODE REVIEW

File: test/form_helper_test.rb [hunk 1]
```diff
@@ -729,6 +729,71 @@
     end
   end
 
+  describe '#text_area' do
+    it "renders " do
```
Comment (joneslee85) [inline test/form_helper_test.rb]: what does it render?

**Comment by joneslee85** [comment]
:+1: I like it

**Comment by AlfonsoUceda** [comment]
Can you add text_area helper to https://github.com/lotus/helpers/blob/master/lib/lotus/helpers/form_helper.rb#L29 ?

:+1:

**Comment by khaiql** [comment]
@AlfonsoUceda, @joneslee85, I added some small changes, please see

File: lib/lotus/helpers/form_helper/form_builder.rb [hunk 2]
```diff
@@ -501,6 +501,36 @@ def file_field(name, attributes = {})
...
+        #   <%=
+        #     # ...
+        #     text_area :hobby, 'Football'
+        #   %>
+        #
+        #   # Output:
+        #   #  <textarea name="user[hobby]" id="user-hobby">Football</textarea>
+        def text_area(name, content = nil, attributes = {})
+          attributes = {name: _input_name(name), id: _input_id(name)}.merge(attributes)
+          _content = content || _value(name)
```
Comment (joneslee85) [inline lib/lotus/helpers/form_helper/form_builder.rb]: @khaiql what do you think about setting default value for `content` argument instead? ie. `def text_area(name, content = _value(name), attributes = {})`

**Comment by joneslee85** [comment]
@khaiql except the minor comment above, :+1:

**Comment by joneslee85** [comment]
@khaiql please squash all commits into 1

**Comment by khaiql** [comment]
@joneslee85 done :D

**Pull request merged**